migration-tools.md

```diff
@@ -13,34 +13,83 @@ This document introduces the user scenarios, advantages, and limitations of thes
-|[Backup & Restore (BR)](/br/backup-and-restore-overview.md) | Migrate a large amount of TiDB custer data by backing up and restoring data. | TiDB| SST, backup.meta files, backup.lock files|<ul><li>Suitable for migrating data to another TiDB cluster</li><li>Support backing up data to an external storage for disaster recovery</li></ul> | <ul><li>When BR restores data to the upstream cluster of TiCDC or Drainer, the restored data cannot be replicated to the downstream by TiCDC or Drainer.</li><li>BR supports operations only between clusters that have the same `new_collations_enabled_on_first_bootstrap` value.</li></ul> |
-| [TiCDC](/ticdc/ticdc-overview.md)| This tool is implemented by pulling TiKV change logs. It can restore data to a consistent state with any upstream TSO, and support other systems to subscribe to data changes.|TiDB | TiDB, MySQL, Kafka, Confluent| Provide TiCDC Open Protocol  | TiCDC only replicates tables that have at least one valid index. The following scenarios are not supported:<ul><li>the TiKV cluster that uses RawKV alone.</li><li>the DDL operation `CREATE SEQUENCE` and the `SEQUENCE` function in TiDB.</li></ul>|
-|[TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md) | Incremental replication between TiDB clusters, such as using one TiDB cluster as the secondary cluster of another TiDB cluster | TiDB | TiDB, MySQL, Kafka, incremental backup files | Support real-time backup and restore. Back up TiDB cluster data to be restored for disaster recovery | TiDB Binlog is incompatible with some TiDB versions. It is recommended that you use [TiCDC](/ticdc/ticdc-overview.md). |
-|[sync-diff-inspector](/sync-diff-inspector/sync-diff-inspector-overview.md) | Comparing data stored in the databases with the MySQL protocol |TiDB, MySQL | TiDB, MySQL| Can be used to repair data in the scenario where a small amount of data is inconsistent | <ul><li>Online check is not supported for data migration between MySQL and TiDB.</li><li>JSON, BIT, BINARY, BLOB and other types of data are not supported.</li></ul> |
+The following list introduces the user scenarios, the supported upstreams and downstreams of migration tools.
```

Comment:
The meaning of this sentence is duplicated with L10. Can we remove it and refine the sentence in L10?

Reply:
updated in [d732e82](https://github.com/pingcap/docs/pull/9580/commits/d732e823e7f7a9bf82dd6ec9cd317e3d62d35160)